sktime/utils/check_imports.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from importlib import import_module
+
+
+def _check_imports(package):
```

Comment:
Since we have to check multiple package in some modules, how about we accept multiple package names and loop over them? 

```python
def _check_soft_dependencies(*packages):
    for package in packages:
        ...
```

Reply:
`check_soft_dependencies` may also be a more meaningful name, would also appreciate a short/informal docstring!